src/api/files.js

```diff
@@ -89,6 +89,14 @@ export function preview(file) {
 	return `${baseURL}/api/files/${file}?alt=media&preview`;
 }
 
+export function previewDocs(fileId) {
+	return `${config.docsUrl}/${fileId}`;
+}
+
+export function isDocumentEdit(type) {
```

Comment:
why did you create this function if it returns the value of another function?
the api folder should only contain functions that calls to the server.
if you need to check if the file can be edited, you can use checkDocumentEdit directly.